### PR TITLE
Support for Laravel 9.x

### PR DESCRIPTION
**Pull request by felixdorn**
This PR adds support for Laravel 9.x, all tests pass. A few depreciations are thrown because `ramsey/collections` and `ramsey/uuid` are not yet up to date.

## Summary
* A few depreciations due to `ramsey/collections` and `ramsey/uuid` not being up to date
* A condition in the `test` cript that runs the test for L9 if we're running PHP8 in development.
* Added an empty `tests/Integration` directory to fix failing tests for PHPUnit >9.

> Maybe consider removing support for 7.2 and 7.3 as they are deprecated.

### CODE REVIEW

**Comment by felixdorn** [comment]
I've submit this PR as #49 and #48 won't be enough to truly support Laravel 9.x in Sidecar.

File: composer.json [hunk 1]
```diff
@@ -11,17 +11,16 @@
...
-        "illuminate/filesystem": "^7.0|^8.0",
-        "illuminate/console": "^7.0|^8.0",
+        "illuminate/filesystem": "^7.0|^8.0|^9.0",
+        "illuminate/console": "^7.0|^8.0|^9.0",
         "maennchen/zipstream-php": "^2.1",
         "guzzlehttp/guzzle": "^7.2",
-        "aws/aws-sdk-php": "^3.20.0"
+        "aws/aws-sdk-php": "^3.20.0",
+        "illuminate/support": "9.*",
+        "orchestra/testbench": "7.*"
```
Comment (dammy001) [inline composer.json:20]: why did you move this to require instead of require-dev

**Comment by felixdorn** [comment]
I think the CI is hanging on PHP 7.x and prefer-lowest.

**Comment by aarondfrancis** [comment]
This looks great! I added a couple of changes and it's all good to go now. Thanks for doing this, the past week has been wild. I appreciate the help! 

> I think the CI is hanging on PHP 7.x and prefer-lowest which is odd...
Yeah it hangs on the old stuff sometimes. I think as a way to make you drop support for it 😂
